libafl/src/stages/mod.rs

```diff
@@ -336,6 +447,193 @@ where
         self.push_stage
             .deinit(fuzzer, state, event_mgr, executor.observers_mut())
     }
+
+    #[inline]
+    fn restart_progress_should_run(&mut self, _state: &mut Self::State) -> Result<bool, Error> {
+        // TODO: Proper restart handling - call post_exec at the right time, etc...
+        Ok(true)
+    }
+
+    #[inline]
+    fn clear_restart_progress(&mut self, _state: &mut Self::State) -> Result<(), Error> {
+        Ok(())
+    }
+}
+
+/// Progress which permits a fixed amount of resumes per round of fuzzing. If this amount is ever
+/// exceeded, the input will no longer be executed by this stage.
+#[derive(Clone, Deserialize, Serialize, Debug)]
+pub struct RetryRestartHelper {
```

Comment:
should we make a 
```
pub trait RestartHelper
 fn restart_progress_should_run(); 
 fn clear_restart_progress();
}
```
trait?

Reply:
It exists, but some helpers need different parameters so the trait doesn't apply